### ISSUE TITLE
Fails to build for target x86_64-unknown-linux-musl

### ISSUE THREAD

**Issue by anowell** [eminence/terminal-size#6]
```
error[E0308]: mismatched types
  --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/terminal_size-0.1.5/src/unix.rs:28:30
   |
28 |         ioctl(STDOUT_FILENO, TIOCGWINSZ, &mut winsize);
   |                              ^^^^^^^^^^ expected i32, found u64
```

Kinda the opposite of issue #3

**Comment by eminence** [comment]
Thanks for the report!  This has been fixed, and the new version has been published v0.1.6

**Comment by anowell** [comment]
Thanks. That was fast!
